config/v1/0000_10_config-operator_01_imagecontentpolicy.crd.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/470
```

Comment:
technically you have to switch this to 874 now

Reply:
This PR looks good to me. Thank you @sttts for helping us get this done. I'll lgtm once this link is fixed.